cpp/ql/test/library-tests/controlflow/guards-ir/ASTGuardsControl.expected

```diff
@@ -74,17 +74,18 @@
 | test.c:109:19:109:23 | ... < ... | false | 113 | 113 |
 | test.c:126:7:126:7 | 1 | true | 126 | 126 |
 | test.c:126:7:126:7 | 1 | true | 126 | 128 |
-| test.c:126:7:126:7 | 1 | true | 131 | 131 |
-| test.c:126:7:126:7 | 1 | true | 131 | 132 |
-| test.c:126:7:126:7 | 1 | true | 134 | 123 |
 | test.c:126:7:126:28 | ... && ... | true | 126 | 128 |
 | test.c:126:12:126:26 | call to test3_condition | true | 126 | 128 |
 | test.c:131:7:131:7 | b | true | 131 | 132 |
-| test.c:137:7:137:7 | 0 | false | 142 | 136 |
+| test.c:137:7:137:7 | 0 | true | 137 | 138 |
+| test.c:137:7:137:7 | 0 | true | 138 | 139 |
```

Comment:
I'm not sure what's going on in the old Guards library, but I think the new results are correct